src/test/java/org/xstefank/TestUtils.java

```diff
@@ -0,0 +1,33 @@
+package org.xstefank;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import org.xstefank.model.yaml.FormatConfig;
+
+import java.io.File;
+import java.io.IOException;
+
+public class TestUtils {
+
+    public static JsonNode testPayload;
+    public static JsonNode badTestPayload;
+    public static FormatConfig formatConfig;
+
+    public static final String YAML = "yaml";
+    private static final String JSON = "json";
+
+    static {
```

Comment:
put this please into a normal method, there is no need to keep it in static block